index.bs

```diff
@@ -55,15 +55,18 @@ A WebDriver [=session=] has a <dfn>WebSocket connection</dfn>, which
 is a network connection, and a <dfn>WebSocket URL</dfn>, which is a
 string. These are both initially null.
 
-When data is recieved on [=WebSocket connection=] it must be processed according to the
-requirements of the [[!RFC6455|WebSockets]] specification. When [=a WebSocket
-message has been received=], an implementation must [=handle an incoming
-message=]. When [=the WebSocket closing handshake is started=] an implementation
-must [=end the session=].
+When data is recieved on [=WebSocket connection=]
+<var>connection</var>, it must be processed according to the
+requirements of the [[!RFC6455|WebSockets]] specification. When [=a
+WebSocket message has been received=], an implementation must [=handle
+an incoming message=] with argument <var>connection</var>. When [=the
+WebSocket closing handshake is started=] an implementation must
+[=close the WebSocket connection=] with argument
+<var>connection</var>.
 
 <div algorithm>
-To <dfn>establish a WebSocket connection</dfn>, with
-argument |session id|, run these steps:
+To <dfn>start listening for a WebSocket connection</dfn>, with
```

Comment:
```suggestion
To <dfn>start listening for a WebSocket connection</dfn>, for a |session|
```

Reply:
Applying this suggestion by itself won't work, but we need that to set "|session|'s WebSocket URL" below.